test_crates/function_has_body/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+pub fn top_level() {}
+
+pub struct Foo;
+
+impl Foo {
+    pub fn inside_impl_block() {}
+}
+
+pub trait Bar {
+    fn trait_no_body();
+
+    fn trait_with_body();
```

Comment:
This function doesn't actually have a body.
```suggestion
    fn trait_with_body() {}
```